### PR TITLE
chore: fix confirmed modal

### PR DESCRIPTION
**Pull request by DjordjeVujatovic**
## Description

- Fix confirmed modal spacing, close button, and twitter -> x
- This PR addresses two Mint Modal related P0 tickets: 

1. https://www.notion.so/lazertechnologies/72fceb443e3842debb9139a62d4ec37a?v=8331767752974fcdae4d0866e80975d0&p=5b7dd786f622465483a3af442f84ce03&pm=s
2. https://www.notion.so/lazertechnologies/72fceb443e3842debb9139a62d4ec37a?v=8331767752974fcdae4d0866e80975d0&p=46474161aaf14ec1aa881aa2c910e666&pm=s

## Ticket(s)

- [69](https://github.com/LazerTechnologies/onchain-summer/issues/69)

## Deploy Notes

Notes regarding deployment of the contained body of work. These should note any
db migrations, nginx routes, infrastructure changes, and anything that must be done before deployment.

- [ ] Need to add environment variables
  - `ENV_VAR=`

## Screenshots (if appropriate)
Screen Shot 2023-08-09 at 4 29 40 PM

### CODE REVIEW

File: src/components/Share/Share.tsx [hunk 1]
```diff
@@ -1,52 +1,71 @@
...
+      <div className="flex flex-col items-center space-x-2">
+        <Link
+          href={tweetUrl}
+          target="_blank"
+          className="flex flex-col items-center"
+        >
+          <TwitterIcon />
+          <p className="text-sm font-mono uppercase text-[#858585] mt-[6px]">
+            X
+          </p>
```
Comment (DjordjeVujatovic) [inline src/components/Share/Share.tsx:40]: This was the only thing changed in this file. The rest of the changes are prettier related

**Review by wilsoncusack — APPROVED** [review]

**Review by wilsoncusack — APPROVED** [review]

**Review by wilsoncusack — APPROVED** [review]

**Review by aflock — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-10 17:46:16 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
Review Error for aflock @ 2023-08-10 17:47:00 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by cb-heimdall** [comment]
Review Error for wilsoncusack @ 2023-08-10 17:50:38 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

**Pull request merged**